src/datumaro/capi/pybind.cpp

```diff
@@ -50,7 +50,12 @@ struct RLE
 
 RLE rleEncode(const byte *M, siz h, siz w)
 {
+
     siz j, k, a = w * h;
+
+    if (a / w != h)
```

Comment:
Shouldn't we also check for cases where 'w' is 0? And if `a/w` results in a floating-point value, we shouldn't compare it using `!=`.

Reply:
@jihyeonyi  w and h would be passed from the size of numpy 2d array so i think it would be verified while initializing 2d array. and this function will not be exposed directly to the python.